es-core/src/resources/TextureData.cpp

```diff
@@ -68,6 +69,7 @@ bool TextureData::initSVGFromMemory(const unsigned char* fileData, size_t length
 	float scale = Math::min(mHeight / svgImage->height, mWidth / svgImage->width);
 	nsvgRasterize(rast, svgImage, 0, 0, scale, dataRGBA, (int)mWidth, (int)mHeight, (int)mWidth * 4);
 	nsvgDeleteRasterizer(rast);
+	nsvgDelete(svgImage);
```

Comment:
Just confirming whether this is safe? From reading the code, even the previous "nsvgDeleteRaterizer" statement worries me, but if it isn't broken...

Reply:
> Just confirming whether this is safe?

[Here's an example usage](https://github.com/memononen/nanosvg/blob/master/example/example2.c) of the nanosvg API. I think it should pretty much follow the pattern, but that's just my evaluation.

>  if it isn't broken...

This is not a critical leak or anything like that, it's just around half a megabyte which isn't that much in this day and age. If you feel uncertain about the patch, we can skip it of course.